tip-distributor/src/claim_mev_workflow.rs

```diff
@@ -55,13 +55,26 @@ pub fn claim_mev_tips(
     let mut transactions = Vec::new();
 
     runtime.block_on(async move {
-        let blockhash = rpc_client
-            .get_latest_blockhash()
-            .await
-            .expect("read blockhash");
-
+        let blockhash = rpc_client.get_latest_blockhash().await.expect("read blockhash");
+        let balance = rpc_client.get_balance(&keypair.pubkey()).await.expect("failed to get balance");
+
+        // heuristic to make sure we have enough funds to cover the rent costs if jito has many validators
+        {
+            let node_count = merkle_trees.generated_merkle_trees.iter().flat_map(|tree| &tree.tree_nodes).count();
+            let min_rent = rpc_client.get_minimum_balance_for_rent_exemption(ClaimStatus::SIZE).await.expect("Failed to calculate min rent");
+            assert!(balance >= node_count as u64 * min_rent);
```

Comment:
we could add in a fee estimation too. like merkle_trees.generated_merkle_trees * tree.tree_nodes * FEE_PER_SIGNATURE

Reply:
this seems perfect to me though, will overestimate but seems fine to alert earlier than later